### PR TITLE
Hf2p Initial Documentation

### PR DESCRIPTION
**Pull request by ghost**
Initial documentation for the Halo Online Engine

### CODE REVIEW

File: src/content/hf2p/readme.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<figure>
+  <a href="HaloOnline_ConceptArt.jpg">
+    <img src="HaloOnline_ConceptArt.jpg" alt="""/>
+  </a>
+  <figcaption>
+    <p>Halo Online's Concept Art</p>
+  </figcaption>
+</figure>
+
+**Halo Online** is a canceled instalment of the Halo game series and runs on a modified [Halo 3][h3] Engine. It was created by [Saber Interactive][saber] and initially released as part of a closed beta in Russia on the PC in 2015-16 by [343 Industries][343i]. In 2016 the game was canceled as a part of *Halo: The Master Chief Collection* on PC was being ported to Microsoft Windows alongside other halo titles in the collection.
```
Comment (num0005) [inline src/content/hf2p/readme.md:10]:
```suggestion
**Halo Online** is a canceled installment of the Halo game series and runs on a modified [Halo 3][h3] Engine. It was created by [Saber Interactive][saber] and initially released as part of a closed beta in Russia on the PC in 2015-16.
```

No evidence that was the reason it was canceled and arguable it was published by Saber Interactive not 343i, should rephrase that to high-light it was a collaboration but that Saber Interactive's platform was used.

File: src/content/hf2p/readme.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+Like other games in the series Halo Online uses Bungie's proprietary [Blam!][engine] and is powered by a highly modified version of the Halo 3 engine, optimized for smooth performance on lower-end PCs.
```
Comment (num0005) [inline src/content/hf2p/readme.md:12]: 343i's engine? Not sure what the guidelines are for the wiki.

File: src/content/hf2p/readme.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+## Eldewrito to MCC: A Legacy (PC, 343 Industries, 2016-Current day)
+During the Halo Online closed testing, a group of modders were able to mod the game and began to work on "ElDewrito", a fan-attempt to bring the classic Halo experience to the PC platform by modding the game to play like the original release of Halo 3. In April 2018, version 0.6 of the mod released to widespread community acclaim, reaching 12,000 concurrent players. Four days following the release of ElDewrito 0.6, the members of the team were contacted by 343 Industries including Frank O'Connor, whereupon they were asked to pause development of the mod, and were not given an official cease and desist or DMCA takedown. An official statement was given by 343 Industries the same day, explaining that the mod was taken down due to legal requirements to protect Microsoft intellectual property.
```
Comment (num0005) [inline src/content/hf2p/readme.md:15]:
```suggestion
During the Halo Online closed testing, a group of modders were able to mod the game and began to work on "ElDewrito", a fan-attempt to bring the classic Halo experience to the PC platform by modding the game to play like the original release of Halo 3.

In April 2018, version 0.6 of the mod released to widespread community acclaim, reaching 12,000 concurrent players. Four days following the release of ElDewrito 0.6, the members of the team were contacted by 343 Industries including Frank O'Connor, whereupon they were asked to pause development of the mod  - however the modding team was not given an official cease and desist or DMCA takedown.
An official statement was given by 343 Industries the same day, explaining that the mod was taken down due to legal requirements to protect Microsoft intellectual property.
```

File: src/content/hf2p/readme.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+On December 15, 2020, it was revealed MCC Season 5: Anvil would include twelve armor permutations that originally came from Halo Online, such as Demo, Suture, and Compass Mjolnir armors.
+
+## PC (Saber Interactive, April 2015)
+This is the first release of Halo Online for PC.
+
+### Modding
+There are several tools and mods for the base game or external programs that users can grab to enhance their experience. Examples include:
+
+* [Halo 3 Editing Kit][h3-ek] - The official editing suite released for the MCC version of Halo 3 largely based on internal tools developed by Bungie and later used by 343 Industries and [Saber Interactive][saber]. Some changes were made to make the tools more user friendly and stable, as well as to support importing content from modern 3D modeling software.
```
Comment (num0005) [inline src/content/hf2p/readme.md:25]: Can't use the H3EK for HO/El-do.

File: src/content/hf2p/readme.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+* [Halo Asset Blender Development Toolset][halo-asset-blender-development-toolset] - A collection of tools for Blender that can be used to export the intermediate files used by the Halo 3 Editing Kit.
```
Comment (num0005) [inline src/content/hf2p/readme.md:26]: Doubt you could use the toolset either but idk what people are doing with tag tool these days.

**Review by num0005 — CHANGES_REQUESTED** [review]

File: src/content/hf2p/readme.md (hunk 5 above)
Comment (0xFA837) [reply src/content/hf2p/readme.md:29]: It's entirely possible to use them for HO/El-do, It's just not documented at all. Feel free to remove them if you feel it's best.

**Comment by 0xFA837** [comment]
All issues should be fixed.

**Review by num0005 — APPROVED** [review]
Seems fine.

**Pull request closed without merging**